components/infobox/wikis/starcraft2/infobox_person_player.lua

```diff
@@ -121,15 +132,15 @@ function StarCraft2Player._getRaceData(race)
 	}
 end
 
-function StarCraft2Player:createBottomContent(infobox)
-	if shouldStoreData then
+function CustomPlayer:createBottomContent(infobox)
+	if _SHOULD_STORE_DATA then
```

Comment:
This if just doesn't make sense, the var should be renamed, because shouldStoreData is not relevant to this function semantically